internal/host/transition.go

```diff
@@ -259,7 +259,7 @@ func (th *transitionHandler) PostResetHost(sw stateswitch.StateSwitch, args stat
 		return errors.New("PostResetHost invalid argument")
 	}
 
-	extra := append(append(make([]interface{}, 0), "StatusUpdatedAt", strfmt.DateTime(time.Now())), resetLogsField...)
```

Comment:
why was that removed?

Reply:
If we change status, this field is set here https://github.com/openshift/assisted-service/blob/3c6c20c6f897a2a0378a6ff41c141028c52b8a9b/internal/host/hostutil/update_host.go#L61
So there is an attempt to change this field twice which causes an error